src/dataloaderinterface/templates/dataloaderinterface/site_registration.html

```diff
@@ -203,6 +201,9 @@
                                                         </tr>
                                                     {% endif %}
                                                 {% endfor %}
+                                                <tr class="no-sensors-row {% if result_form|length > 0 %}hidden{% endif %}">
```

Comment:
Like I mentioned above, adding the `bg-white` CSS class here would solve the problem of the row highlighting when hovering over an empty list.

Reply:
Agree.